astrology.lic

```diff
@@ -15,15 +15,27 @@ class Astrology
     @ap_training = settings.astral_plane_training['train_in_ap']
     @ap_source = settings.astral_plane_training['train_source']
     @ap_destination = settings.astral_plane_training['train_destination']
+    @pools = {
+      'lore' => 0,
+      'magic' => 0,
+      'survival' => 0,
+      'offensive combat' => 0,
+      'defensive combat' => 0,
+      'future events' => 0
+    }
 
     do_buffs(settings)
 
-    check_astral
     train_astrology
   end
 
   def do_buffs(settings)
     settings.astrology_buffs['spells'].each do |spell|
+      if spell['abbrev'].casecmp('rtr').zero?
```

Comment:
Does this mean RTR will only be cast once an hour, even if the buff expires before that?